File: WebApp4.Mvc.Project/Models/Employee.cs
using System.Web;

namespace WebApp4.Mvc.Project.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("HumanResources.Employee")]
    public partial class Employee
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Employee()
        {
            EmployeeDepartmentHistories = new HashSet<EmployeeDepartmentHistory>();
            EmployeePayHistories = new HashSet<EmployeePayHistory>();
            JobCandidates = new HashSet<JobCandidate>();
            PurchaseOrderHeaders = new HashSet<PurchaseOrderHeader>();
            ImagePath = "~/AppFiles/Images/default.jpg";
        }

        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int BusinessEntityID { get; set; }

        [Required(ErrorMessage = "This Field is required.")]
        [StringLength(15)]
        [DisplayName("Id")]
        public string NationalIDNumber { get; set; }

        [Required(ErrorMessage = "This Field is required.")]
        [StringLength(256)]
        [DisplayName("Login")]
        public string LoginID { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public short? OrganizationLevel { get; set; }

        [Required(ErrorMessage = "This Field is required.")]
        [StringLength(50)]
        [DisplayName("Job")]
        public string JobTitle { get; set; }

        [Column(TypeName = "date")]
        [DisplayName("Birth Date")]
        public DateTime BirthDate { get; set; }

        [Required(ErrorMessage = "This Field is required.")]
        [StringLength(1)]
        public string MaritalStatus { get; set; }

        [Required(ErrorMessage = "This Field is required.")]
        [StringLength(1)]
        public string Gender { get; set; }

        [Column(TypeName = "date")]
        public DateTime HireDate { get; set; }

        public bool SalariedFlag { get; set; }

        public short VacationHours { get; set; }

        public short SickLeaveHours { get; set; }

        public bool CurrentFlag { get; set; }

        public Guid rowguid { get; set; }

        public DateTime ModifiedDate { get; set; }

        public virtual Person Person { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<EmployeeDepartmentHistory> EmployeeDepartmentHistories { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<EmployeePayHistory> EmployeePayHistories { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<JobCandidate> JobCandidates { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<PurchaseOrderHeader> PurchaseOrderHeaders { get; set; }

        public virtual SalesPerson SalesPerson { get; set; }

        public string ImagePath { get; set; }

        [NotMapped]
        public HttpPostedFileBase ImageUpload { get; set; }



    }
}

File: WebApp4.Mvc.Project/Scripts/Employee/employee.js

const param = {
    "EndDate": "2022-05-05",
    "IdEmployee": 23,
    "Name": "Desde la Web",
    "StartDate": "2022-05-05"
}

const fillDataTable = () => {    
    $('#employee-list').DataTable({
        deferRender: true,
        scrollX: true,
        scrollY: 500,        
        'ajax': {
            'url': 'list-param',
            'type': "GET",
            'contentType': 'application/json',
            'data': param
        },
        'columns': [
            { data: 'BusinessEntityID' },
            { data: 'NationalIDNumber' },
            { data: 'LoginID' },
            { data: 'OrganizationLevel' },
            { data: 'JobTitle' },
            { data: 'BirthDate' },
            { data: 'MaritalStatus' },
            { data: 'Gender' },
            { data: 'HireDate' },
            { data: 'SalariedFlag' },
            { data: 'VacationHours' },
            { data: 'SickLeaveHours' },
            { data: 'CurrentFlag' },
            { data: 'rowguid' },
            { data: 'ModifiedDate' },
            { data: 'ImagePath' }
        ]
    });
}

const employeeList = () => {
    $.ajax({
        url: "Employee/List",
        success: (result) => {
            return result
        }
    });
}



